batched/dense/impl/KokkosBatched_Vector_SIMD_Arith.hpp

```diff
@@ -105,7 +105,7 @@ static KOKKOSKERNELS_SIMD_ARITH_RETURN_TYPE(float, 4) operator+(const Vector<SIM
 KOKKOS_FORCEINLINE_FUNCTION
 static KOKKOSKERNELS_SIMD_ARITH_RETURN_TYPE(double, 4) operator+(const Vector<SIMD<double>, 4> &a,
                                                                  const Vector<SIMD<double>, 4> &b) {
-  double4 r_val;
+  KOKKOSKERNELS_SIMD_ARITH_RETURN_TYPE(double, 4)::data_type r_val;
```

Comment:
Would we perhaps have preferred the following for consistency with other code in the same file?

```suggestion
  Vector<SIMD<double>, 4>::data_type r_val;
```

And, if we would, should we do the same replacement for the `float2`, `double2` and `float4` specializations as well for consistency?